lib/bootloader_setup.pm

```diff
@@ -35,17 +36,32 @@ our @EXPORT = qw(
   tianocore_select_bootloader
 );
 
+# prevent grub2 timeout; 'esc' would be cleaner, but grub2-efi falls to the menu then
+sub stop_grub_timeout {
```

Comment:
Now you can see also future 😆 
Well in this kind of ideology I will force you to merge nice pieces of code